src/components/Resume.vue

```diff
@@ -0,0 +1,127 @@
+        h5 Work Experience
+      .col-12#work(v-for="(work, index) in resume.work")
+        .row
+          .col-md-6.text-left
+            h6 {{work.company}}
+          .col-md-6.text-right
+            h6 {{work.startDate}} - {{work.endDate}}
+          .col-md-6.text-left {{work.position}}
+          .col-md-6.text-right
+            a(:href="work.website") {{work.website}}
+          .col-12.text-left {{work.summary}}
+          .col-md-4(v-for="(highlight, index) in work.highlights")
+            | {{highlight}}
+    .row#subtitle(v-if=("resume.volunteer.length > 0"))
+      .col-12.text-left#subtitle
+        h5 Volunteer Experience
+      .col-12#volunteer(v-for="(volunteer, index) in resume.volunteer")
+        .row
+          .col-md-6.text-left
+            h6 {{volunteer.organization}}
+          .col-md-6.text-right
+            h6 {{volunteer.startDate}} - {{volunteer.endDate}}
+          .col-md-6.text-left {{volunteer.position}}
+          .col-md-6.text-right
+            a(:href="volunteer.website") {{volunteer.website}}
+          .col-12.text-left {{volunteer.summary}}
+          .col-6(v-for="(highlight, index) in volunteer.highlights")
+            | {{highlight}}
+    .row#subtitle(v-if=("resume.education.length > 0"))
+      .col-12.text-left#subtitle
+        h5 Education
+      .col-12#education(v-for="(education, index) in resume.education")
+        .row
+          .col-md-6.text-left
+            h6 {{education.institution}}
+          .col-md-6.text-right
+            h6 {{education.startDate}} - {{education.endDate}}
+          .col-md-6.text-left {{education.area}}, {{education.studyType}}
+          .col-md-6.text-right {{education.gpa}}
+          .col-md-4(v-for="(course, index) in education.courses")
+            | {{course}}
+    .row#subtitle(v-if=("resume.awards.length > 0"))
+      .col-12.text-left#subtitle
+        h5 Awards
+      .col-md-6#awards(v-for="(award, index) in resume.awards")
+        .row
+          .col-md-6.text-left
+            h6 {{award.title}}
+          .col-md-6.text-right
+            h6 {{award.date}}
+          .col-md-6.text-left {{award.awarder}}
+          .col-md-6.text-right {{award.summary}}
+    .row#subtitle(v-if=("resume.references.length > 0"))
+      .col-12.text-left#subtitle
+        h5 Interests
+      .col-3#interests(v-for="(interest, index) in resume.interests")
+        .row
+          .col-md-12
+            h6 {{interest.name}}
+            section(v-for="(keyword, index) in interest.keywords")
+              | {{keyword}}
+    .row#subtitle(v-if=("resume.references.length > 0"))
+      .col-12.text-left#subtitle
+        h5 References
+      .col-6#reference(v-for="(reference, index) in resume.references")
+        .row
+          .col-md-6.text-left
+            h6 {{reference.name}}
+            | {{reference.reference}}
+    .row#subtitle(v-else)
+      .col-12.text-right#subtitle
+        | references aviable upon request.
+</template>
```

Comment:
Leaving this all in one document makes it relatively hard to read. Is there a way to keep it concise like it is here, but in separate files?

Reply:
Man, Pug makes this so easy compared to JSX

Reply:
it really is so nice, only problem comes when copying code from sites(if you want to convert it)

Reply:
As for the one document it was just to keep it contained for transferring to my website
https://derekjohnston.ca/projects/MintbeanHackathons